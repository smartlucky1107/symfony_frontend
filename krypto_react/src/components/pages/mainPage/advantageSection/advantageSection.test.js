import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import AdvantageSection from "./advantageSection";

afterEach(cleanup);

describe("Advantage Section component", () => {
    it("Should correctly render component", () => {
        const { asFragment } = render(<AdvantageSection />);

        expect(asFragment()).toMatchSnapshot();
    });
});
