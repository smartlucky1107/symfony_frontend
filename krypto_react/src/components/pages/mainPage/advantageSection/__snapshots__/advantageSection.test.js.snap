// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Advantage Section component Should correctly render component 1`] = `
<DocumentFragment>
  <section
    class="container advantageSection"
  >
    <header>
      <h1
        class="dashBrdHeader"
      >
        <span>
           
          <strong
            class="blue"
          />
           
        </span>
      </h1>
    </header>
    <div
      class="content"
    />
  </section>
</DocumentFragment>
`;
